lms/static/js/student_account/models/user_account_model.js

```diff
@@ -27,7 +27,7 @@
             },
 
             parse : function(response) {
-                if (_.isNull(response)) {
+                if (_.isNull(response) || _.isUndefined(response)) {
```

Comment:
As mentioned before, I don't want to proliferate this sort of check throughout all our parse methods unless it is possible for response to be null and/or undefined outside of unit tests.